src/main/java/com/CoachPlan/dto/StudentDTO.java

```diff
@@ -1,13 +1,24 @@
 package com.CoachPlan.dto;
 
+
 public class StudentDTO {
 	
 	private String loginID;
+	public String fullName;
+	public String Position;
+	public String Sport;
 	//TODO what other fields should a student have? Password field as well? 
 	
+	public String getPosition() {
+		return this.Position;
+	}
+	
+	public void setName(String name) {
+		this.fullName = name;
+	}
 	
-	public void setLoginID(String login) {
```

Comment:
See note above... if you're keeping the attribute, might as well keep the setter method, too.